docs/src/main/paradox/home.md

```diff
@@ -13,6 +13,21 @@ The examples in this documentation use
 
 Release notes are found at [Github releases](https://github.com/akka/reactive-kafka/releases).
 
+If you want to try out a connector that has not yet been released, give @ref[snapshots](snapshots.md) a spin which are published after every commit on master.
+
+
+Matching Versions
+-----------------
+
+|Kafka  | Akka version | Alpakka Kafka Connector
+|-------|--------------|-------------------------
+|1.1.x  | 2.5.x        | [release 0.20](https://github.com/akka/reactive-kafka/releases)
+|1.0.x  | 2.5.x        | [release 0.20](https://github.com/akka/reactive-kafka/releases)
+|0.11.x | 2.5.x        | [release 0.19](https://github.com/akka/reactive-kafka/milestone/19?closed=1)
+|0.11.x | 2.4.x        | [release 0.18](https://github.com/akka/reactive-kafka/milestone/18?closed=1)
+
```

Comment:
I think that is a good move. And since this is now in the home page of the documentation, we can remove it from the README.md.